### DISCUSSION TITLE
Hack for YI Home Camera 3

### DISCUSSION THREAD

**Discussion by cbroissa-github** [roleoroleo/yi-hack-Allwinner-v2#177] [Q&A]
Hello,
I have this model:
YI Home Camera 3
YYS.2518
IFUSY3228
Fw: 8.3.0
There is a hack for this?
I think it’s a recent model, I don’t understand why the firmware is not above 9.x, but I can’t update it.
Thanks.

**Comment by roleoroleo** [comment]
Check this hack:
https://github.com/roleoroleo/yi-hack-Allwinner

**Reply by cbroissa-github** [reply]
I have already tried the y25ga but after one hour the blue light was always blinking.

**Reply by cbroissa-github** [reply]
If you want, I can put some logs to create a new hack for this cam?
